### PR TITLE
fix: font warning

### PR DESCRIPTION
**Pull request by mangkoran**
Hello! Long time no see.

I found the following warning when I tried to compile v1.0.1.

```
❯ typst compile template.typ
warning: Typst's default font has changed from Linux Libertine to its successor Libertinus Serif
    ┌─ @preview/imprecv:1.0.1/cv.typ:106:20
    │
106 │     #set text(font: uservars.bodyfont, weight: "medium", size: uservars.fontsize * 1)
    │                     ^^^^^^^^^^^^^^^^^
    │
    = hint: please set the font to `"Libertinus Serif"` instead

warning: unknown font family: consolas
    ┌─ @preview/imprecv:1.0.1/cv.typ:372:39
    │
372 │             #set text(size: 5pt, font: "Consolas", fill: silver)
    │                                        ^^^^^^^^^^

warning: Typst's default font has changed from Linux Libertine to its successor Libertinus Serif
  ┌─ @preview/imprecv:1.0.1/cv.typ:6:14
  │
6 │         font: uservars.bodyfont,
  │               ^^^^^^^^^^^^^^^^^
  │
  = hint: please set the font to `"Libertinus Serif"` instead

warning: Typst's default font has changed from Linux Libertine to its successor Libertinus Serif
   ┌─ @preview/imprecv:1.0.1/cv.typ:44:24
   │
44 │         #set text(font: uservars.headingfont, size: 1.5em, weight: "bold")
   │                         ^^^^^^^^^^^^^^^^^^^^
   │
   = hint: please set the font to `"Libertinus Serif"` instead

warning: Typst's default font has changed from Linux Libertine to its successor Libertinus Serif
   ┌─ @preview/imprecv:1.0.1/cv.typ:31:24
   │
31 │         #set text(font: uservars.headingfont, size: 1em, weight: "bold")
   │                         ^^^^^^^^^^^^^^^^^^^^
   │
   = hint: please set the font to `"Libertinus Serif"` instead
```

This PR changes Linux Libertine to Libertinus Serif and Consolas to Deja Vu Sans Mono. Also I do some little touch to the template docs/comments as some variables already quite literal what they do.

Please let me know if you have question and suggestion. Thank you!

### CODE REVIEW

**Comment by mangkoran** [comment]
@jskherman Friendly mention :)

File: cv.typ [hunk 1]
```diff
@@ -377,7 +377,7 @@
     place(
         bottom + right,
         block[
-            #set text(size: 5pt, font: "Consolas", fill: silver)
+            #set text(size: 5pt, font: "Deja Vu Sans Mono", fill: silver)
```
Comment (jskherman) [inline cv.typ:380]: Please kindly change to `DejaVu Sans Mono` exactly if this is the proper font name (just basing on what is written in the [Typst Docs](https://typst.app/docs/reference/text/text/#parameters-font))

**Review by jskherman — CHANGES_REQUESTED** [review]
All good, please just remove the extra space for the monospace `DejaVu Sans Mono` font and it's good to go 👍.

File: cv.typ (hunk 1 above)
Comment (mangkoran) [reply cv.typ:380]: There are inconsistencies in typst repo so thank you for pointing that 😅

image

**Pull request merged**